src/scripts/browserDeviceProfile.js

```diff
@@ -371,7 +371,7 @@ import browser from './browser';
                                     || videoTestElement.canPlayType('video/mp4; codecs="avc1.640029, mp3"').replace(/no/, '');
 
         // Not sure how to test for this
-        const supportsMp2VideoAudio = browser.edgeUwp || browser.tizen || browser.web0s;
+        const supportsMp2VideoAudio = browser.edgeUwp || browser.tizen || browser.web0s || (supportsMp3VideoAudio ? browser.chrome || browser.edgeChromium || browser.firefox : '');
```

Comment:
Maybe split this?
```suggestion
        let supportsMp2VideoAudio = browser.edgeUwp || browser.tizen || browser.web0s;

        // If browser supports MP3, it presumably supports MP2 as well
        if (supportsMp3VideoAudio && (browser.chrome || browser.edgeChromium || browser.firefox)) {
            supportsMp2VideoAudio = true;
        }
```

_At some point, I'd like to add the ability to pass support status in `options` as here:_
https://github.com/jellyfin/jellyfin-web/blob/1cc64865459fb669907022d30e3e7c2c8c177352/src/scripts/browserDeviceProfile.js#L435-L443

Also, are you sure this is true for all versions of browser?

Reply:
I've added what you requested and tested support in browsers.
Firefox 83 is the first version where mp2 files play, it didn't work in earlier ones.
Chrome 49 is the earliest version I tried and it played fine.